internal/libhive/docker.go

```diff
@@ -187,33 +188,29 @@ func (b *dockerBackend) StopContainer(containerID string) error {
 }
 
 // CreateNetwork creates a docker network.
-func (b *dockerBackend) CreateNetwork(name string) (networkID string, err error) {
-	// list networks to make sure not to duplicate
-	existing, err := b.client.ListNetworks()
-	if err != nil {
-		return "", err
-	}
-	// check for existing networks with same name, and if exists, remove
-	for _, exists := range existing {
-		if exists.Name == name {
-			if err := b.client.RemoveNetwork(exists.ID); err != nil {
-				return "", err
-			}
+func (b *dockerBackend) CreateNetwork(name string) (string, error) {
+	existing, err := b.NetworkNameToID(name)
```

Comment:
I'm not sure about this. I think it should just fail if the network already exists.